### DISCUSSION TITLE
自动生成的目录重复了咋整？

### DISCUSSION THREAD

**Discussion by Zephery** [vuepress/ecosystem#143] [Q&A] [answered]
<img width="388" alt="image" src="https://github.com/vuepress/ecosystem/assets/14806207/3090f90f-6c8b-43e8-bbdd-528954997d3a">

**Comment by Mister-Hope** [comment]
A reproduction is needed

**Comment by Zephery** [comment]
首页目录重复：https://wenzhihuai.com/  
代码：https://github.com/Zephery/MyWebsite/blob/master/docs/README.md?plain=1

config的配置：https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/config.ts  
navbar配置：https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/navbar.ts  
主题配置：https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/theme.ts  
感谢大佬帮忙看看~

**Reply by Mister-Hope** [reply]
upgrade to latest version first.

**Comment by Zephery** [comment, marked as answer]
确实更新到最新版就好了....

**Comment by Zephery** [comment]
又出了个新问题，当下拉到文章某个子标题的时候，刷新会出现404
https://wenzhihuai.com/java/%E5%9F%BA%E4%BA%8Ekubernetes%E7%9A%84%E5%88%86%E5%B8%83%E5%BC%8F%E9%99%90%E6%B5%81.html#%E5%9B%9B%E3%80%81%E6%80%A7%E8%83%BD%E5%8E%8B%E6%B5%8B

停在某目录：
<img width="1431" alt="image" src="https://github.com/vuepress/ecosystem/assets/14806207/0ce4d54e-24cc-4fc8-8cea-e834f3d4ead0">

刷新后：
<img width="1410" alt="image" src="https://github.com/vuepress/ecosystem/assets/14806207/58f984a3-c923-4f07-adab-63e4081f5e3b">

config的配置：https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/config.ts
navbar配置：https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/navbar.ts
主题配置：https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/theme.ts
感谢大佬帮忙看看~

**Reply by pengzhanbo** [reply]
此问题已在 https://github.com/vuepress/core/issues/1561 中提出，
并提了 PR https://github.com/vuepress/core/pull/1562 等待修复发版。
